code/game/objects/structures/dropship_ammo.dm

```diff
@@ -26,6 +26,8 @@
 	var/warning_sound = 'sound/machines/hydraulics_2.ogg'
 	var/ammo_used_per_firing = 1
 	var/point_cost = 0 //how many points it costs to build this with the fabricator, set to 0 if unbuildable.
+	///Type of ammo
+	var/ammo_type_used
```

Comment:
Change it back on /obj/structure/ship_ammo I told you to do it BECAUSE they were named the same

Reply:
Then say that and don't be cryptic; as a rule I don't have time to guess.